script.module.pyrollbar/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyrollbar"
+       name="pyrollbar"
+       version="0.15.2"
+       provider-name="rollbar, Lee Smith">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.22.0" optional="false"/>
+    <import addon="script.module.six" version="1.15.0" optional="false"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Error tracking and logging from Python to Rollbar</summary>
+    <description lang="en_GB">Python notifier for reporting exceptions, errors, and log messages to Rollbar.</description>
+    <platform>all</platform>
+    <license>MIT</license>
+    <source>https://github.com/rollbar/pyrollbar</source>
```

Comment:
please change this to your upstream (as the module is patched). After that it's good to go.

Reply:
Done 👍